docs/WritingTests.asciidoc

```diff
@@ -133,7 +133,7 @@ An example usage:
 
 [source,perl]
 -------------------------------------------------------------------
-sub test_flags () {
+sub test_flags ($) {
```

Comment:
Are those actually needed? AFAIK the modules are not loaded with full "signatures" support so this would be detected as prototype, not signature, isn't it?

Reply:
I guess it depends, I create my new custom class as
```
  package mybase;
  use Mojo::Base 'basetest', -signatures;
```

and then was surprised with

`Too many arguments for subroutine 'mybase::test_flags' (got 1; expected 0) at /usr/lib/os-autoinst/autotest.pm line 239.`

Reply:
Also it is difference between no signatures

```
sub test_flags { {fatal => 1} }
```

and what we have in documentation
```
sub test_flags () { {fatal => 1} }
```

right?

Reply:
Note that prototypes are ignored for method calls, and we call it as `$self->test_flags();`. So normally it doesn't make any difference what we specify there.

If you use signatures, that's of course a different thing. But then a `($)` signature I wouldn't consider helpful, it should rather be `($basetest)` or something.

Maybe we should rather go with no prototype / signature in the documentaion by default, and add a section about signatures.